cli/src/ui.rs

```diff
@@ -130,7 +131,35 @@ impl UiOutput {
                 }
             }
             UiOutput::BuiltinPaged { pager } => {
-                pager.finalize();
+                if let Err(minus_error) = pager.finalize() {
+                    writeln!(
+                        ui.error_with_heading("Error"),
```

Comment:
nit: use `ui.warning_default()` (and `warning_no_heading()`) as it's not a fatal error

Reply:
I might use two separate `warning_default()`-s, since they warn about different things.

Reply:
Also, thanks!